shell/platform/fuchsia/dart_runner/integration/meta/dart_aot_runner_test.cmx

```diff
@@ -3,14 +3,12 @@
         "data": "data/dart_aot_runner_test"
     },
     "sandbox": {
-        "features": [
-            "deprecated-ambient-replace-as-executable"
-        ],
         "services": [
             "fuchsia.cobalt.LoggerFactory",
             "fuchsia.fonts.Provider",
             "fuchsia.logger.LogSink",
             "fuchsia.modular.Clipboard",
+            "fuchsia.modular.ContextWriter",
```

Comment:
I believe this is deprecated and not needed. but we can remove that later if this is still in topaz.

Reply:
yeah, letting it be for now.